lib/main.dart

```diff
@@ -1,10 +1,14 @@
 import 'package:flutter/material.dart';
+import 'package:shared_preferences/shared_preferences.dart';
```

Comment:
Can we use [`flutter_secure_storage`](https://pub.dev/packages/flutter_secure_storage) instead? Looks like that's the "correct" way to store sensitive app data.